.nvmrc

```diff
@@ -1 +1 @@
-18.17
+20.6.0
```

Comment:
Seems like it should be a separate PR?

Reply:
Yes it can be, I am just taking the change from this https://github.com/bigcommerce/checkout-js/pull/1933 and applying it. Seems an easy and straightforward one.

Reply:
For anyone watching, we decided to move this to a different PR